src/bridge/cockpitpcpmetrics.c

```diff
@@ -47,14 +47,24 @@ typedef struct {
 } MetricInfo;
 
 typedef struct {
+  int context;
+  gint64 start;
+} ArchiveInfo;
```

Comment:
Because pmlogger will create a new archive every time it starts and I think it is a nice service of the metrics channel to put them back together again.